### ISSUE TITLE
dimension check 3D spline

### ISSUE THREAD

**Issue by mlenel** [jacobwilliams/bspline-fortran#23]
Hi Jacob,

I think there is a typo in the dimension check for the 3D splines, line 1708 in the current sub module.

It says
```fortran
if (size(z)/=size(f3,2)) then; iflag = 702; return; end if
```

but should say
```fortran
if (size(z)/=size(f3,3)) then; iflag = 702; return; end if
```

Thanks for all your work on this, so useful!
Moritz

**Comment by jacobwilliams** [comment]
Thanks! Probably a copy-and-paste bug. I will fix this tonight.

**Comment by jacobwilliams** [comment]
Fixed in the [5.0.1](https://github.com/jacobwilliams/bspline-fortran/releases/tag/5.0.1) release.

**Comment by mlenel** [comment]
Thank you!
